### ISSUE TITLE
Can't launch plugin

### ISSUE THREAD

**Issue by heismauri** [nkomarn/extra-compostables#1]
Hello there, I loved the idea of your plugin but sadly I cannot use it on PaperSpigot 1.81.1

This is the error I get when launching my server
```
13.01 17:42:07 [Server] INFO Enabling extracompostables v1.0.1
13.01 17:42:07 [Server] ERROR Error occurred while enabling extracompostables v1.0.1 (Is it up to date?)
13.01 17:42:07 [Server] INFO java.lang.NoSuchMethodError: 'java.lang.Object net.minecraft.core.RegistryBlocks.get(net.minecraft.resources.MinecraftKey)'
13.01 17:42:07 [Server] INFO at xyz.nkomarn.extracompostables.ExtraCompostables.registerCompostable(ExtraCompostables.java:57) ~[ExtraCompostables.jar:?]
13.01 17:42:07 [Server] INFO at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?]
13.01 17:42:07 [Server] INFO at xyz.nkomarn.extracompostables.ExtraCompostables.onEnable(ExtraCompostables.java:27) ~[ExtraCompostables.jar:?]
13.01 17:42:07 [Server] INFO at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
13.01 17:42:07 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
13.01 17:42:07 [Server] INFO at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
13.01 17:42:07 [Server] INFO at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:564) ~[paper-1.18.1.jar:git-Paper-148]
13.01 17:42:07 [Server] INFO at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:478) ~[paper-1.18.1.jar:git-Paper-148]
13.01 17:42:07 [Server] INFO at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:727) ~[paper-1.18.1.jar:git-Paper-148]
13.01 17:42:07 [Server] INFO at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:503) ~[paper-1.18.1.jar:git-Paper-148]
13.01 17:42:07 [Server] INFO at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:313) ~[paper-1.18.1.jar:git-Paper-148]
13.01 17:42:07 [Server] INFO at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1202) ~[paper-1.18.1.jar:git-Paper-148]
13.01 17:42:07 [Server] INFO at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-148]
13.01 17:42:07 [Server] INFO at java.lang.Thread.run(Thread.java:833) ~[?:?]
13.01 17:42:07 [Server] INFO Disabling extracompostables v1.0.1
```

**Comment by nkomarn** [comment]
The plugin isn't currently compatible with 1.18.1, but I can push an update within a few hours!

**Comment by heismauri** [comment]
OMG That would be aaawesome! Can't wait!

**Comment by nkomarn** [comment]
resolved by 6124ec69ed92ee40f0b7290a3cc52192e732275d
